src/spring-cloud/azext_spring_cloud/_validators_enterprise.py

```diff
@@ -75,3 +99,73 @@ def _is_valid_profile_name(profile):
 def _is_valid_app_and_profile_name(pattern):
     parts = pattern.split('/')
     return len(parts) == 2 and _is_valid_app_name(parts[0]) and _is_valid_profile_name(parts[1])
+
+
+def validate_buildpacks_binding_name(namespace):
+    if not _is_valid_buildpacks_binding_name_pattern(namespace.name):
+        raise CLIError("Buildpacks Binding name should follow pattern {}"
+                       .format(BUILDPACKS_BINDING_NAME_REGEX_PATTTERN))
+
+
+def _is_valid_buildpacks_binding_name_pattern(name):
+    return match(BUILDPACKS_BINDING_NAME_REGEX_PATTTERN, name) is not None
+
+
+def validate_buildpacks_binding_properties(namespace):
+    """ Extracts multiple space-separated properties in key[=value] format """
+    if isinstance(namespace.properties, list):
+        properties_dict = {}
+        for item in namespace.properties:
+            properties_dict.update(validate_tag(item))
+        namespace.properties = properties_dict
+
+
+def validate_buildpacks_binding_secrets(namespace):
+    """ Extracts multiple space-separated secrets in key[=value] format """
+    if isinstance(namespace.secrets, list):
+        secrets_dict = {}
+        for item in namespace.secrets:
+            secrets_dict.update(validate_tag(item))
+        namespace.secrets = secrets_dict
+
+
+def enterprise_only_and_binding_not_exist(cmd, namespace):
+    only_support_enterprise(cmd, namespace)
```

Comment:
Since service name already validates it, here can skip it to save network overhead

Reply:
There is no validator in
```python
service_name_type = CLIArgumentType(options_list=['--service', '-s'], help='Name of Azure Spring Cloud, you can configure the default service using az configure --defaults spring-cloud=<name>.', configured_default='spring-cloud')
```
And,  I've tried this
### Codes
##### _params.py
```python
    for scope in ['spring-cloud build-service buildpacks-binding create',
                  'spring-cloud build-service buildpacks-binding show',
                  'spring-cloud build-service buildpacks-binding delete']:
        with self.argument_context(scope) as c:
            c.argument('service', service_name_type, validator=only_support_enterprise)

    with self.argument_context('spring-cloud build-service buildpacks-binding create') as c:
        c.argument('service', service_name_type, validator=enterprise_only_and_binding_not_exist)
```
##### buildpacks_binding.py
```
def buildpacks_binding_create(cmd, client, resource_group, service,
                              name, type, properties=None, secrets=None):
    print("hello buildpacks_binding_create")
    exit(0)
```
#####  Validators
```
def only_support_enterprise(cmd, namespace):
    print("hello only_support_enterprise")
    return
...
def enterprise_only_and_binding_not_exist(cmd, namespace):
    print("hello enterprise_only_and_binding_not_exist")
    return
```
### Output
---
Only one validator takes effect.
```
(CLIDev0617) PS C:\devops\CLIEnvs\CLIDev0617> az spring-cloud build-service buildpacks-binding create -n my-binding -g jiec-enterprise --service jiec-e-asc --type ApplicationInsights
hello enterprise_only_and_binding_not_exist
hello buildpacks_binding_create
```

Reply:
I do believe this is because you add validators on `service` twice. If you type `spring-cloud build-service buildpacks-binding show`, it will invoke `only_support_enterprise`